pkg/watch/health.go

```diff
@@ -0,0 +1,257 @@
+package watch
+
+import (
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/square/p2/pkg/health"
+	"github.com/square/p2/pkg/kp"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/preparer"
+	"github.com/square/p2/pkg/util/net"
+)
+
+// These constants should probably all be something the p2 user can set
+// in their preparer config...
+
+// number of milliseconds between reality store checks
+const POLL_KV_FOR_PODS = 2000
+
+// number of milliseconds between health checks
+const HEALTHCHECK_INTERVAL = 1000
+
+// TTL on healthchecks in seconds
+const TTL = 60
```

Comment:
I'd feel more comfortable if we multiplied these by their units during initialization instead of trusting all the clients to multiply correctly:

```
const POLL_KV_FOR_PODS = 2 * time.Second
const HEALTHCHECK_INTERVAL = 1 * time.Second
const TTL = 60 * time.Second
```